package.json

```diff
@@ -138,8 +138,8 @@
     "metascraper-logo": "^5.25.7",
     "metascraper-logo-favicon": "^5.25.7",
     "metascraper-title": "^5.25.7",
-    "mocha": "^8.3.2",
-    "mocha-junit-reporter": "^2.0.0",
+    "mocha": "^10.2.0",
```

Comment:
This is a major upgrade from mocha v8 to mocha v10. Are there any breaking changes in mocha v9 and mocha v10 that might affect this project?

Reply:
They had drop Node.js v12.x support & Internet-Explorer-11 support & drop  AMD/RequireJS support & drop Growl notification support,import-first loading of test files. I think at develop we don't need node<= 12.x and AMD support. Mocha as devDependencies is ok

Reply:
https://github.com/mochajs/mocha/blob/master/CHANGELOG.md
All  Breaking Changes is here

Reply:
ESLint still supports Node.js v12.x, and we're running tests on it:

https://github.com/eslint/eslint/blob/7190d98ff40023f24b0c6a98319ae8a82c99ff5b/.github/workflows/ci.yml#L37-L42

Test runs on this PR seem fine at the moment, but since mocha v10 doesn't support Node.js v12.x this could break in the future.

Reply:
Node 12.x had end of life, should we drop it？

Reply:
> This is unit test result on node12.x [#16653](https://github.com/eslint/eslint/actions/runs/3674127172/jobs/6215318490)

Yes, it works at the moment, but if `mocha` decides to use some features that are not available in Node 12, that could break our testing on Node 12.

> Node 12.x had end of life, should we drop it？

We can't drop any versions of Node.js in non-major versions of ESLint. I think the best course of action would be to keep `mocha` version we're using now until the next major version of ESLint.

Reply:
Agree with you, please kill this pull request open until next major version